automation/taskcluster/docker/build.dockerfile

```diff
@@ -100,6 +100,9 @@ RUN curl -L https://dl.google.com/android/repository/android-ndk-${ANDROID_NDK_V
 ENV ANDROID_NDK_TOOLCHAIN_DIR /root/.android-ndk-r15c-toolchain
 ENV ANDROID_NDK_API_VERSION 21
 
+# The android gradle plugin uses $ANDROID_NDK_HOME rather than $ANDROID_NDK_ROOT.
+ENV ANDROID_NDK_HOME /build/android-ndk
+
```

Comment:
@eoger IIUC, this change to the dockerfile will be picked up automagically by the taskcluster logic, without us having to explicitly build and publish a new image. Accurate?

Reply:
Yup